adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/uap/PlatformInformation.cs

```diff
@@ -37,14 +37,17 @@
 using Windows.System.UserProfile;
 using Microsoft.Identity.Core;
 using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.OAuth2;
+using Windows.System;
+using System.Collections.Generic;
+using Windows.Foundation.Collections;
 
 namespace Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Platform
 {
     internal class PlatformInformation : PlatformInformationBase
```

Comment:
>PlatformInformation [](start = 19, length = 19)

would it make sense to have a base for Uap? Not sure why some of this is different for adal vs. msal

Reply:
Yes and no. I agree that platform information could use a refactoring as it is too complex and has no tests (mainly because of multi-targetting). However, this is possibly not the best time to touch it before release. A base UAP class is non trivial because there already exists an inheritance hierarchy.

Reply:
ok